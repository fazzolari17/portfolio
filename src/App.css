@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600&family=Lexend+Deca:wght@600;700&family=Ubuntu:wght@300;400;500;700&display=swap');

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

/* Fonts */
/* font-family: 'Inter', sans-serif; */
/* font-family: 'Ubuntu', sans-serif; */
/* font-family: 'Lexend Deca', sans-serif; */


/* :root.light, 
:root {
  --header-background: #071013;
  --header-contrastColor: #75ABBC;
  --hero-background: #395E66;
  --header-text: #F0F8FF;
} */


/* ============================ */
/* ========== Logo ============ */
/* ============================ */
.logo {
  font-family: 'Lexend Deca', sans-serif;
  font-size: 3rem;
  
}

.tilt {
  display: inline-block;
  position: absolute;
  top: 25px;
  left: 40px;
  transform: scale(1.25);
  color: var(--header-constrastColor, #75ABBC);
}

/* ========================================== */
/* ========== Home Page Image Grid ========== */
/* ========================================== */

.gallery {
    display: grid;
    grid-template-columns: repeat(5, 1fr);
    /* grid-template-rows: repeat(4,); */
    grid-template-rows: auto auto;
    grid-gap: 5px;
    padding: 5px;
    background-color: var(--hero-background, #395E66);
    margin-top: 60px;
}

.gallery img {
    width: 100%;
    height: 100%;
    object-fit: cover;
}

.span-1 {
  grid-column: span 1;
  grid-row: span 1;
}

.span-2 {
  grid-column: span 2;
  grid-row: span 1;
}

.span-3 {
  grid-column: span 2;
  grid-row: span 2;
}
