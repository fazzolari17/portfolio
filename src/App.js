import './App.css';
import Header from './components/header/Header';
import Home from './components/home/Home';
import Contact from './components/contact/Contact';
import Projects from './components/projects/Projects';
import AboutMe from './components/aboutMe/AboutMe';
import Footer from './components/footer/Footer';
import ProjectDetail from './components/projects/ProjectDetail';
import MobileHeader from './components/header/MobileHeader';

import {
  BrowserRouter as Router,
  Routes, Route
} from 'react-router-dom';

import useViewport from './hooks/useViewport';

function App() {
  const { isMobile } = useViewport();

  return (
    <main>
      <Router>
        {isMobile ?  <MobileHeader /> : <Header />}
  
        <Routes>
          <Route path='/projects/:id' element={<ProjectDetail />} />
          <Route path='/projects' element={<Projects />} />
          <Route path='/aboutMe' element={<AboutMe />} />
          <Route path='/contact' element={<Contact/>} />
          <Route path='/' element={<Home />} />
        </Routes>

      </Router>
      <Footer />
    </main>
  );
}

export default App;
