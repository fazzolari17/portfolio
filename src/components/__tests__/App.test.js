import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
// import renderer from 'react-test-renderer';
import App from '../../App';

describe('App', () => {
  // describe('Snapshot', () => {
  //   it('renders correctly', () => {
  //     const tree = renderer.create(
  //       <App />
  //     ).toJSON();

  //     expect(tree).toMatchSnapshot();
  //   });
  // });
  describe('Main', () => {
    beforeEach(() => {
      render(<App />);
    });

    it('renders the page and dispays menu', () => {
      const home = screen.getByText(/home/i);
      const aboutMe = screen.getByText(/about me/i);
      const projects = screen.getByText(/projects/i);
      const contact = screen.getByText(/contact/i);
      expect(home).toBeInTheDocument();
      expect(aboutMe).toBeInTheDocument();
      expect(projects).toBeInTheDocument();
      expect(contact).toBeInTheDocument();
    });

    it('renders the page and dispays footer with correct icons', () => {
      const footerText = screen.getByText(/designed & created by: giuseppe fazzolari/i);
      const githubIcon = screen.getByLabelText('github link');
      const linkedInIcon = screen.getByLabelText(/linkedin link/i);
      const emailIcon = screen.getByLabelText(/email link/i);

      expect(footerText).toBeInTheDocument();
      expect(githubIcon).toBeInTheDocument();
      expect(linkedInIcon).toBeInTheDocument();
      expect(emailIcon).toBeInTheDocument();
    });

  });
});