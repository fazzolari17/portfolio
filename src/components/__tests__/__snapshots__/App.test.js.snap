// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot renders correctly 1`] = `
<main>
  <header
    data-testid="header"
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "var(--header-background, #071013)",
        "color": "var(--header-text, #F0F8FF)",
        "display": "flex",
        "fontFamily": "Inter, sans-serif",
        "fontWeight": "500",
        "height": "60px",
        "justifyContent": "space-between",
        "left": 0,
        "listStyle": "none",
        "padding": "1em",
        "position": "fixed",
        "textDecoration": "none",
        "textTransform": "uppercase",
        "top": 0,
        "width": "100vw",
        "zIndex": "10",
      }
    }
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <div
        className="logo"
        data-testid="logoG"
      >
        G
        <span
          className="tilt"
          data-testid="logoF"
        >
          F
        </span>
      </div>
    </a>
    <nav
      style={
        Object {
          "marginRight": "2rem",
        }
      }
    >
      <a
        href="/"
        onClick={[Function]}
        style={
          Object {
            "borderBottom": "2px solid var(--header-text, #F0F8FF)",
            "color": "var(--header-text, #F0F8FF)",
            "listStyle": "none",
            "marginLeft": "2em",
            "textDecoration": "none",
            "textTransform": "uppercase",
          }
        }
      >
        Home
      </a>
      <a
        href="/aboutMe"
        onClick={[Function]}
        style={
          Object {
            "color": "var(--header-text, #F0F8FF)",
            "listStyle": "none",
            "marginLeft": "2em",
            "textDecoration": "none",
            "textTransform": "uppercase",
          }
        }
      >
        About Me
      </a>
      <a
        href="/projects"
        onClick={[Function]}
        style={
          Object {
            "color": "var(--header-text, #F0F8FF)",
            "listStyle": "none",
            "marginLeft": "2em",
            "textDecoration": "none",
            "textTransform": "uppercase",
          }
        }
      >
        Projects
      </a>
      <a
        href="/contact"
        onClick={[Function]}
        style={
          Object {
            "color": "var(--header-text, #F0F8FF)",
            "listStyle": "none",
            "marginLeft": "2em",
            "textDecoration": "none",
            "textTransform": "uppercase",
          }
        }
      >
        Contact
      </a>
    </nav>
  </header>
  <div
    className="gallery"
  >
    <img
      alt="sun set over the horizon"
      className="span-2"
      src="https://images.unsplash.com/photo-1486912500284-6f2462ba07ea?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="brown leafed tree near bed of yellow flowers"
      className="span-1"
      src="https://images.unsplash.com/photo-1421930866250-aa0594cea05c?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="snow-covered mountain during a twilight sky"
      className="span-2"
      src="https://images.unsplash.com/photo-1502657877623-f66bf489d236?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="scenery of mountain canyon"
      className="span-2"
      src="https://images.unsplash.com/photo-1479030160180-b1860951d696?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="green trees"
      className="span-2"
      src="https://images.unsplash.com/photo-1494280986787-c49b328829dd?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="closeup photo of yellow sunflowers"
      className="span-1"
      src="https://images.unsplash.com/photo-1504567961542-e24d9439a724?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="birds eye view photography of cloudy sky"
      className="span-2"
      src="https://images.unsplash.com/photo-1493130952181-47e36589f64d?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="scenery of a grassfield during sunset"
      className="span-1"
      src="https://images.unsplash.com/uploads/14121010130570e22bcdf/e1730efe?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="rainbow near green grass ranges"
      className="span-2"
      src="https://images.unsplash.com/photo-1463743981180-c90485a4f274?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="ocean near trees and rocks"
      className="span-2"
      src="https://images.unsplash.com/photo-1462400362591-9ca55235346a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="silhouette of person standing on rock surrounded by body of water"
      className="span-2"
      src="https://images.unsplash.com/photo-1490730141103-6cac27aaab94?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="ice bergs and alp mountains facing calm body of water"
      className="span-1"
      src="https://images.unsplash.com/photo-1494564605686-2e931f77a8e2?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="brown tree trunks lot"
      className="span-2"
      src="https://images.unsplash.com/photo-1505567745926-ba89000d255a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="sand landscape"
      className="span-1"
      src="https://images.unsplash.com/photo-1499988921418-b7df40ff03f9?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="seashore"
      className="span-2"
      src="https://images.unsplash.com/photo-1533371452382-d45a9da51ad9?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="green leaf trees under blue sky"
      className="span-2"
      src="https://images.unsplash.com/photo-1469827160215-9d29e96e72f4?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="lavender field"
      className="span-2"
      src="https://images.unsplash.com/photo-1499002238440-d264edd596ec?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="landscape photo of brown mountain"
      className="span-1"
      src="https://images.unsplash.com/photo-1489619243109-4e0ea59cfe10?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="brown rock formation under blue sky"
      className="span-2"
      src="https://images.unsplash.com/photo-1508739773434-c26b3d09e071?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="white sand"
      className="span-1"
      src="https://images.unsplash.com/photo-1559666126-84f389727b9a?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="photography of mountain during sunset"
      className="span-2"
      src="https://images.unsplash.com/photo-1433477077279-9354d2d72f6b?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="brown wooden dock between lavender flower field near body of water during golden hour"
      className="span-2"
      src="https://images.unsplash.com/photo-1532274402911-5a369e4c4bb5?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="lake with mountain reflections"
      className="span-2"
      src="https://images.unsplash.com/photo-1580137189272-c9379f8864fd?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="trees under cloudy sky during sunset"
      className="span-1"
      src="https://images.unsplash.com/photo-1470252649378-9c29740c9fa8?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="worms eye view of forest during day time"
      className="span-2"
      src="https://images.unsplash.com/photo-1440342359743-84fcb8c21f21?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="orange flowers"
      className="span-1"
      src="https://images.unsplash.com/photo-1465146344425-f00d5f5c8f07?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="island during daytime"
      className="span-2"
      src="https://images.unsplash.com/photo-1566419808810-658178380987?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="closeup photo of green leafed plants"
      className="span-2"
      src="https://images.unsplash.com/photo-1463107971871-fbac9ddb920f?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="landscape photography of brown mountain"
      className="span-2"
      src="https://images.unsplash.com/photo-1431794062232-2a99a5431c6c?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
    <img
      alt="rule of thirds photography of pink and white lotus flower floating on body of water"
      className="span-1"
      src="https://images.unsplash.com/photo-1474557157379-8aa74a6ef541?crop=entropy&cs=tinysrgb&fit=max&fm=jpg&ixid=MnwzOTk5MDR8MHwxfHJhbmRvbXx8fHx8fHx8fDE2NzM5MjM3ODg&ixlib=rb-4.0.3&q=80&w=400"
    />
  </div>
  <div
    style={
      Object {
        "display": "flex",
        "height": "100vh",
        "left": 0,
        "position": "fixed",
        "top": 0,
        "width": "100vw",
      }
    }
  >
    <section
      style={
        Object {
          "alignSelf": "center",
          "backgroundColor": "var(--hero-background, #395E66)",
          "borderRadius": "5px",
          "boxShadow": "",
          "color": "var(--header-text, #F0F8FF)",
          "display": "flex",
          "flexDirection": "column",
          "fontFamily": "Lexend Deca, sans-serif",
          "justifySelf": "center",
          "lineHeight": "1.6",
          "margin": "2rem auto",
          "maxWidth": "50%",
          "padding": "2rem",
        }
      }
    >
      Fix the cause, not the symptom.
      <div
        style={
          Object {
            "alignSelf": "flex-end",
            "justifySelf": "flex-end",
            "maxWidth": "fit-content",
          }
        }
      >
        - Steve Maguire
      </div>
    </section>
  </div>
  <footer
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "var(--header-background,#071013)",
        "bottom": "0",
        "color": "var(--header-text ,#F0F8FF)",
        "display": "flex",
        "fontFamily": "Inter, sans-serif",
        "fontWeight": "700",
        "height": "75px",
        "justifyContent": "center",
        "left": "0",
        "right": "0",
      }
    }
  >
    <div
      style={
        Object {
          "alignItems": "center",
          "display": "flex",
          "justifyContent": "center",
        }
      }
    >
      <p
        style={
          Object {
            "marginRight": "1rem",
          }
        }
      >
        Designed & Created By: Giuseppe Fazzolari ©
      </p>
      <br />
      <div>
        <a
          aria-label="github link"
          href="https://github.com/fazzolari17"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            color="var(--header-text, #F0F8FF)"
            fill="currentColor"
            height="2rem"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "var(--header-text, #F0F8FF)",
                "marginRight": "1rem",
              }
            }
            viewBox="0 0 448 512"
            width="2rem"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M400 32H48C21.5 32 0 53.5 0 80v352c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48V80c0-26.5-21.5-48-48-48zM277.3 415.7c-8.4 1.5-11.5-3.7-11.5-8 0-5.4.2-33 .2-55.3 0-15.6-5.2-25.5-11.3-30.7 37-4.1 76-9.2 76-73.1 0-18.2-6.5-27.3-17.1-39 1.7-4.3 7.4-22-1.7-45-13.9-4.3-45.7 17.9-45.7 17.9-13.2-3.7-27.5-5.6-41.6-5.6-14.1 0-28.4 1.9-41.6 5.6 0 0-31.8-22.2-45.7-17.9-9.1 22.9-3.5 40.6-1.7 45-10.6 11.7-15.6 20.8-15.6 39 0 63.6 37.3 69 74.3 73.1-4.8 4.3-9.1 11.7-10.6 22.3-9.5 4.3-33.8 11.7-48.3-13.9-9.1-15.8-25.5-17.1-25.5-17.1-16.2-.2-1.1 10.2-1.1 10.2 10.8 5 18.4 24.2 18.4 24.2 9.7 29.7 56.1 19.7 56.1 19.7 0 13.9.2 36.5.2 40.6 0 4.3-3 9.5-11.5 8-66-22.1-112.2-84.9-112.2-158.3 0-91.8 70.2-161.5 162-161.5S388 165.6 388 257.4c.1 73.4-44.7 136.3-110.7 158.3zm-98.1-61.1c-1.9.4-3.7-.4-3.9-1.7-.2-1.5 1.1-2.8 3-3.2 1.9-.2 3.7.6 3.9 1.9.3 1.3-1 2.6-3 3zm-9.5-.9c0 1.3-1.5 2.4-3.5 2.4-2.2.2-3.7-.9-3.7-2.4 0-1.3 1.5-2.4 3.5-2.4 1.9-.2 3.7.9 3.7 2.4zm-13.7-1.1c-.4 1.3-2.4 1.9-4.1 1.3-1.9-.4-3.2-1.9-2.8-3.2.4-1.3 2.4-1.9 4.1-1.5 2 .6 3.3 2.1 2.8 3.4zm-12.3-5.4c-.9 1.1-2.8.9-4.3-.6-1.5-1.3-1.9-3.2-.9-4.1.9-1.1 2.8-.9 4.3.6 1.3 1.3 1.8 3.3.9 4.1zm-9.1-9.1c-.9.6-2.6 0-3.7-1.5s-1.1-3.2 0-3.9c1.1-.9 2.8-.2 3.7 1.3 1.1 1.5 1.1 3.3 0 4.1zm-6.5-9.7c-.9.9-2.4.4-3.5-.6-1.1-1.3-1.3-2.8-.4-3.5.9-.9 2.4-.4 3.5.6 1.1 1.3 1.3 2.8.4 3.5zm-6.7-7.4c-.4.9-1.7 1.1-2.8.4-1.3-.6-1.9-1.7-1.5-2.6.4-.6 1.5-.9 2.8-.4 1.3.7 1.9 1.8 1.5 2.6z"
            />
          </svg>
        </a>
        <a
          aria-label="linkedin link"
          href="https://www.linkedin.com/in/giuseppe-fazzolari"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            color="var(--header-text, #F0F8FF)"
            fill="currentColor"
            height="2rem"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "var(--header-text, #F0F8FF)",
                "marginRight": "1rem",
              }
            }
            viewBox="0 0 448 512"
            width="2rem"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
            />
          </svg>
        </a>
        <a
          aria-label="email link"
          href="mailto:fazzolari17@gmail.com"
          rel="noreferrer"
          target="_blank"
        >
          <svg
            color="var(--header-text, #F0F8FF)"
            fill="none"
            height="2rem"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": "var(--header-text, #F0F8FF)",
                "marginRight": "1rem",
              }
            }
            viewBox="0 0 24 24"
            width="2rem"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M3 8l7.89 5.26a2 2 0 002.22 0L21 8M5 19h14a2 2 0 002-2V7a2 2 0 00-2-2H5a2 2 0 00-2 2v10a2 2 0 002 2z"
              strokeLinecap="round"
              strokeLinejoin="round"
              strokeWidth="2"
            />
          </svg>
        </a>
      </div>
    </div>
  </footer>
</main>
`;
